README-legacy.md

```diff
@@ -1,4 +1,7 @@
-noahong is a fork https://github.com/JDonner/NoAho
+# Legacy README
+
+
+## Differences with `NoAho`
```

Comment:
Not sure why, but this isn't rendered as a title: https://github.com/victorbenichoux/noahong/blob/readme-format/README-legacy.md